src/Map.ts

```diff
@@ -202,7 +203,13 @@ export class Map extends maplibregl.Map {
     });
 
     this.primaryLanguage = options.language ?? config.primaryLanguage;
-    this.secondaryLanguage = config.secondaryLanguage;
+    this.forceLanguageUpdate =
+      this.primaryLanguage === Language.STYLE ||
+      this.primaryLanguage === Language.STYLE_LOCK
+        ? false
+        : true;
```

Comment:
```suggestion
    this.forceLanguageUpdate =
      this.primaryLanguage === Language.STYLE ||
      !(this.primaryLanguage === Language.STYLE_LOCK)
```

nit

Reply:
Definitely not equivalent expressions. Maybe what you had in mind was that instead:
```ts
this.forceLanguageUpdate =
      !(this.primaryLanguage === Language.STYLE ||
      this.primaryLanguage === Language.STYLE_LOCK);
```